### PR TITLE
#24: Update Converse to 6.0.0

### PR DESCRIPTION
**Pull request by mnsuccess**
A new version of Converse was released. This PR  makes use of that version  (converse 6.0.0).
https://github.com/igniterealtime/openfire-inverse-plugin/issues/24

### CODE REVIEW

File: changelog.html [hunk 1]
```diff
@@ -43,7 +43,73 @@ <h1>
     inVerse Plugin Changelog
 </h1>
 
+<<<<<<< HEAD
```
Comment (guusdk) [inline changelog.html]: These symbols (`<<<<` and `====`) should not be in this file. I think you tried to merge a conflict. That seems to have gone wrong.

Also, there now are two headers: '5.0.5 Release 1' and '6.0.0 Release 1' right under each-other. That's not right.

File: changelog.html [hunk 2]
```diff
@@ -43,7 +43,73 @@ <h1>
...
+            <li>_converse.api.chats.get() now only returns one-on-one chats, not the control box or headline notifications.</li>
+            <li>The show_only_online_users setting has been removed.</li>
+            <li>_converse.api.alert.show is now _converse.api.show and instead of taking an integer for the type, "info", "warn" or "error" should be passed in.</li>
+            <li>The converse-headline plugin has been split up into converse-headlines and converse-headlines-view.</li>
+        </ul>
+    </li>
+</ul>
+
+<p><b>5.0.5 Release 1</b> -- November 20, 2019</p>
+>>>>>>> #24 update version and changelog
```
Comment (guusdk) [inline changelog.html]: This line is also an artifact of a merge. It should be removed.

**Comment by mnsuccess** [comment]
@guusdk The new release 6.0.1 of converse doesn't have a distribution package in the Asset. Do I have to Generate a builds with the code source of release 6.0.1   in order to get the dist package for 6.0.1?

**Comment by guusdk** [comment]
@mnsuccess Isn't this what you need? 
image

**Comment by mnsuccess** [comment]
> @mnsuccess Isn't this what you need?
> image



Screenshot 2020-03-13 at 14 48 29
it's weird :)

**Comment by guusdk** [comment]
Whaaat? Works for me, even in an incognito tab in my browser. Try reloading / using a different browser maybe?

**Comment by mnsuccess** [comment]
> Whaaat? Works for me, even in an incognito tab in my browser. Try reloading / using a different browser maybe?
> 
> Update: only now seeing that you edited your screenshot. :-D

I have just reloaded the browser now I can see the asset.

**Comment by mnsuccess** [comment]
@guusdk  Done . could you review and test? need to close this PR as soon as possible :)

File: pom.xml [hunk 3]
```diff
@@ -8,7 +8,7 @@
     </parent>
     <groupId>org.igniterealtime.openfire.plugins</groupId>
     <artifactId>inverse</artifactId>
-    <version>6.0.0.0-SNAPSHOT</version>
+    <version>6.0.1-SNAPSHOT</version>
```
Comment (guusdk) [inline pom.xml]: I think that this should be `6.0.1.1-SNAPSHOT` (6.0.1 Release 1)

**Comment by guusdk** [comment]
Thanks for all of your hard work!

**Pull request merged**